site/src/utils/slug.js

```diff
@@ -64,3 +70,7 @@ export const makeSessionSlugProcessor = (preserveUnicode = false) => {
 		return slug;
 	}
 }
+
+export const makeSlugProcessor = (preserveUnicode = false) => {
```

Comment:
Is there a reason for this to be its own function and for it to be exported?

Reply:
No, I was planning to use it in another part of the code where I wanted to slugify a single string not in relation to a list of others, but I decided to wait for that, reverted.